src/Types/ScriptLookups.purs

```diff
@@ -2,15 +2,19 @@ module Types.ScriptLookups
   ( MkUnbalancedTxError(..)
```

Comment:
I'm OK with the solution you have for this, but I wanted to ask what you think of also exporting some `unsafe...` versions of lookups that need to call `validatorHash`, etc...?

Reply:
It ultimately boils to how unsafe 
```
exports._scriptHashFromBytesImpl = maybe => bytes => {
    return hashFromImpl(CardanoWasm.ScriptHash.from_bytes)(maybe)(bytes);
};
```
is. Is this related to freeing allocated CSL objects? I guess it would just error out if it doesn't work right so it should be okay?